README.md

```diff
@@ -343,315 +375,340 @@ convert asff2hdf            Translate a AWS Security Finding Format JSON into a
+                              Vulnerabilities Scan.
+  USAGE
+    $ saf convert netsparker2hdf -i <netsparker-xml> -o <hdf-scan-results-json> [-h]
+
+  FLAGS
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) Input Netsparker XML File
+    -o, --output=<value>  (required) Output HDF JSON File
 
   EXAMPLES
-    saf convert netsparker2hdf -i netsparker_results.xml -o output-hdf-name.json
+    $ saf convert netsparker2hdf -i netsparker_results.xml -o output-hdf-name.json
 ```
 
-
 ##### Nikto to HDF
-
 ```
-convert nikto2hdf           Translate a Nikto results JSON file into a Heimdall
-                            Data Format JSON file
-  OPTIONS
-    -i, --input=input          Input Nikto Results JSON File
-    -o, --output=output        Output HDF JSON File
+convert nikto2hdf             Translate a Nikto results JSON file into a Heimdall
+                              Data Format JSON file.
+                              Note: Currently this mapper only supports single
+                              target Nikto Scans
+  USAGE
+    $ saf convert nikto2hdf -i <nikto-json> -o <hdf-scan-results-json> [-h]
+
+  FLAGS
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) Input Niktop Results JSON File
+    -o, --output=<value>  (required) Output HDF JSON File
 
   EXAMPLES
-    saf convert nikto2hdf -i nikto-results.json -o output-hdf-name.json
+    $ saf convert nikto2hdf -i nikto-results.json -o output-hdf-name.json
 ```
 
 ##### Prisma to HDF
-
 ```
-convert prisma2hdf         Translate a Prisma Cloud Scan Report CSV file into a Heimdall Data Format
-                          JSON file
-  OPTIONS
-    -i, --input=input          Prisma Cloud Scan Report CSV
-    -o, --output=output        Output HDF JSON File
+convert prisma2hdf            Translate a Prisma Cloud Scan Report CSV file into
+                              Heimdall Data Format JSON files
+  USAGE
+    $ saf convert prisma2hdf -i <prisma-cloud-csv> -o <hdf-output-folder> [-h]
 
-  DESCRIPTION
-    Note: This mapper generates multiple output files, named by corresponding host names
+  FLAGS
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) Prisma Cloud Scan Report CSV
+    -o, --output=<value>  (required) Output HDF JSON file
 
   EXAMPLES
-    saf convert prisma2hdf -i prisma.csv -o output-hdf-name.json
+    $ saf convert prisma2hdf -i prismacloud-report.csv -o output-hdf-name.json
 ```
 
 ##### Prowler to HDF
-
 ```
-convert prowler2hdf         Translate a Prowler-derived AWS Security Finding
-                            Format results from concatenated JSON blobs into a
-                            Heimdall Data Format JSON file
-  OPTIONS
-    -i, --input=input          Input Prowler ASFF JSON File
-    -o, --output=output        Output HDF folder
+convert prowler2hdf           Translate a Prowler-derived AWS Security Finding
```

Comment:
thought we fixed the readme a bit ago - concatenated json blobs are JSONL/JSON Lines files.  please change any references to this instead.